config/services.yaml

```diff
@@ -112,4 +112,11 @@ services:
         arguments:
             $url: '%env(APP_GITLAB_API_URL)%'
 
+    Buddy\Repman\Service\Doctrine\PackageSynchronizedListener:
```

Comment:
honestly, I think I prefer to call it explicitly in the handler than such a hidden listener